### DISCUSSION TITLE
part-ii/2018-99/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#303] [Question comments]
Part II 2018 Number Theory: Paper 1, Section I, G 

https://questions.tripos.org/part-ii/2018-99/

**Comment by callumhobbis** [comment]
<details>
<summary>Sketch solution</summary>

**(a):** Bookwork.

**(b):** I looked at the system of congruences $x+i\equiv p_i\enspace(\text{mod }p_i^2)$ for $i=1,\ldots,1000$ and $p_i$ distinct primes (e.g. take $p_i$ to be the $i^\text{th}$ prime), and noted that this has a solution $n$ by the CRT. Then $n+i$ has $p_i\mid n+i$ but $p_i^2\nmid n+i$.

</details>
